group__enums.html

```diff
@@ -4726,9 +4726,9 @@ <h2 class="memtitle"><span class="permalink"><a href="#gab084d3a6d6ddd144ea6df4a
 </td></tr>
 <tr><td class="fieldname"><a id="ggab084d3a6d6ddd144ea6df4af965f93fca1fd9f2d48998b0eb519f4aef9e26784e"></a>DAMAGE_POOP&#160;</td><td class="fielddoc"><p>damage from red poop </p>
 </td></tr>
-<tr><td class="fieldname"><a id="ggab084d3a6d6ddd144ea6df4af965f93fca62cf902c707d3c6e4c4d128610348845"></a>DAMAGE_DEVIL&#160;</td><td class="fielddoc"><p>damage comes from devil room deal </p>
+<tr><td class="fieldname"><a id="ggab084d3a6d6ddd144ea6df4af965f93fca62cf902c707d3c6e4c4d128610348845"></a>DAMAGE_DEVIL&#160;</td><td class="fielddoc"><p> </p>
 </td></tr>
-<tr><td class="fieldname"><a id="ggab084d3a6d6ddd144ea6df4af965f93fca02369e68e3333e33c832c31096a4f8d2"></a>DAMAGE_ISSAC_HEART&#160;</td><td class="fielddoc"><p>Indicates the damage has been redirected from <a class="el" href="namespace_isaac.html">Isaac</a>'s Heart familiar </p>
+<tr><td class="fieldname"><a id="ggab084d3a6d6ddd144ea6df4af965f93fca02369e68e3333e33c832c31096a4f8d2"></a>DAMAGE_ISSAC_HEART&#160;</td><td class="fielddoc"><p>indicates the damage has been redirected from <a class="el" href="namespace_isaac.html">Isaac</a>'s Heart familiar </p>
```

Comment:
Please revert this change. All lines need to start with a capital letter

Reply:
But this is the only one that doesn't in the entire list. I did it for consistency. Do I make all others with a capital as well?

Reply:
Yes please :)

Reply:
Done.